app/controllers/users.js

```diff
@@ -0,0 +1,17 @@
+const logger = require('../logger');
+const { User } = require('../models');
+const { hashPassword } = require('../helpers');
+
+exports.signUp = (req, res, next) =>
+  hashPassword(req.body)
```

Comment:
Be careful using req.body directly, because in this way i can insert any field of user and break something in DB (like delete_at, updated_at, etc)